java/pom.xml

```diff
@@ -637,7 +637,7 @@
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.11</version>
+        <version>4.12</version>
```

Comment:
Did you need this version change or was it an unassociated change? As with fixing the imports, you don't want to tie random changes into a single patch.